src/gui/transferlistwidget.cpp

```diff
@@ -818,6 +820,21 @@ void TransferListWidget::applyToSelectedTorrents(const std::function<void (BitTo
     }
 }
 
+bool TransferListWidget::torrentContainsPreviewableFiles(const BitTorrent::TorrentHandle *const torrent) const
```

Comment:
The function does not use `this`, which means it has to be either a static method or a free function (which I would prefer, as it is just a simple helper).

Reply:
>it has to be either a static method

It's overthinking too. Static method makes sense if it needs to access private members of some class instance.